apps/playground-web/src/app/insight/insightBlueprints.ts

```diff
@@ -0,0 +1,187 @@
+// This file is auto-generated by the update-insight-blueprints script
+import type { MinimalBlueprintSpec } from "./utils";
+
+export const insightBlueprints: MinimalBlueprintSpec[] = [
```

Comment:
we can't generate these from the downloaded files? would be generated when running the script i guess

Reply:
This file is auto generated